app/src/main/java/com/example/fittracker/mvp/presenters/MainActivityPresenter.java

```diff
@@ -11,7 +15,7 @@ public MainActivityPresenter(LogInContract.View view, LogInContract.Model model)
 
     @Override
     public void onLogInPressed() {
-        if (mModel.isValidLogIn(new User(mView.getEmail(), mView.getPassword()))) {
+        if (mModel.isValidLogIn(new User(mView.getEmail(), mView.getPassword(), EMPTY, EMPTY))) {
```

Comment:
instead of sending EMPTY for those parameters, you can create a constructor only with email and password

Reply:
Done